workflows/workflows_templates/cerbos/policies/workflow_run.yaml.j2

```diff
@@ -0,0 +1,32 @@
+# Auto-generated by running 'make codegen'. Do not edit.
+# Make changes to the template codegen/templates/cerbos/policies/class_name.yaml.j2 instead.
```

Comment:
Tiny nit: can we update the comment with the path for the override template?

Reply:
Done!